src/Nethermind/Nethermind.Init/Steps/InitializeBlockchain.cs

```diff
@@ -158,7 +159,15 @@ protected virtual Task InitBlockchain()
             return Task.CompletedTask;
         }
 
-        protected virtual ITransactionProcessor CreateTransactionProcessor(CodeInfoRepository codeInfoRepository, IVirtualMachine virtualMachine, IWorldState worldState)
+        private void WarmupEvm()
+        {
+            IKeyValueStoreWithBatching db = new MemDb();
+            TrieStore trieStore = new(new NodeStorage(db), No.Pruning, Persist.EveryBlock, new PruningConfig(), NullLogManager.Instance);
+            WorldState state = new(trieStore, db, NullLogManager.Instance);
```

Comment:
I recommend `IWorldStateManager.CreateResettableWorldState` or something, set the root to `Keccak.EmptyTreeHash`. Way to create world state may change depending on type of statedb.